README.md

```diff
@@ -13,12 +13,15 @@ You will need the following things properly installed on your computer.
 
 * [Git](http://git-scm.com/)
 * [Node.js](http://nodejs.org/) (with NPM)
+* [Yarn](https://yarnpkg.com/)
 * [Bower](http://bower.io/)
 * [Ember CLI](http://ember-cli.com/)
 * [PhantomJS](http://phantomjs.org/)
 
 ## Installation
-* `npm install`
+
+* `git clone` this repository
+* `yarn`
```

Comment:
Notes:

1. For new user facing instructions, we try to avoid vague shorthand in favor of explicit commands fully written out (eg `yarn install` instead of `yarn`)

2. Per flowdock discussion, please note that Travis CI will need to run the command with an optional flag as follows: `yarn install --pure-lockfile` ([ref](https://github.com/yarnpkg/yarn/issues/570))

3. When I ran `yarn install` without the above flag, on this branch, I got changes to my lockfile. Can you be sure the file is regenerated against the results of a clean `node_modules` directory?